rosdistro_additional_recipes.yaml

```diff
@@ -71,3 +71,17 @@ moveit2_tutorials:
   rev: 27b5ed0232cd08de9631d26e823dccc6e8766513
   url: https://github.com/moveit/moveit2_tutorials.git
   version: 0.1.0
+
+# TODO: Remove orbbec_* packages when they are available from ros2-gbp
+orbbec_camera:
+  tag: release/jazzy/orbbec_camera/1.5.13-1
```

Comment:
I tagged a new release based on the latest main, hopefully it's not an issue

Reply:
Would it be possible to tag as 1.5.12.1 instead of 1.5.13 ? This would avoid any conflict with any future 1.5.13 release.